src/vs/workbench/contrib/notebook/browser/notebookBrowser.ts

```diff
@@ -224,7 +224,7 @@ export interface INotebookEditor {
 	/**
 	 * Focus the container of a cell (the monaco editor inside is not focused).
 	 */
-	focusNotebookCell(cell: ICellViewModel, focusEditor: boolean): void;
+	focusNotebookCell(cell: ICellViewModel, focusEditor: boolean, focusOutput?: boolean): void;
```

Comment:
we may want to change the second parameter to an enum `cellContainer | editor | output` `focusEditor: true, focusOutput: true` won't make sense

Reply:
It was enough

Reply:
You're right. That is probably the best way to do this.